module1/build tools/core/src/main/java/com/epam/core/Utils.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.core;
+
+import com.epam.task1.StringUtils;
+import java.util.Arrays;
+
+public class Utils {
+
+    public static void main(String[] args) {
```

Comment:
Whycdo you need main method inside this class?

Reply:
Main netthod is required to create jar file.